lib/src/sql/number.rs

```diff
@@ -308,6 +309,16 @@ impl Number {
 			Number::Decimal(v) => v.round(precision as i64).into(),
 		}
 	}
+
+	pub fn pow(self, power: Number) -> Number {
+		match (self, power) {
+			(Number::Int(v), Number::Int(p)) if p >= 0 && p < u32::MAX as i64 => {
+				Number::Int(v.pow(p as u32))
+			}
+			(v, p) => Number::Float(v.as_float().pow(p.as_float())),
+			// TODO: Number::Decimal?
```

Comment:
We'll need to add this functionality for Decimals....

Reply:
This is now implemented for Decimal.pow(Integer), but I don't know if we can do it for `Decimal . pow ( Float / Decimal )`

We could round the floats to be integers, same with the BigDecimal - but it won't be precise or pretty.

Reply:
@mathe42 Awesome! Didn't know that that's how it could be done.

Unfortunately, `ln` isn't available. `exp` seems to be available though.

Reply:
@EtienneBruines Thats how you define `a**b` in math you define `exp` (as an infinite sum) and `ln` (as the inverse of `exp`) and then say `a**b` is defined as `exp(ln(a)*b)`.